config/jobs/kubernetes/sig-scalability/sig-scalability-presubmit.yaml

```diff
@@ -0,0 +1,333 @@
+        - --
+        - --build=bazel
+        - --cluster=
+        - --env=HEAPSTER_MACHINE_TYPE=n1-standard-8
+        - --env-file=jobs/env/ci-kubernetes-e2e-scalability-common.env
+        - --env-file=jobs/env/ci-kubernetes-e2e-scalability-highspeed-common.env
+        - --extract=local
+        - --gcp-nodes=2000
+        - --gcp-project=k8s-scale-testing
+        - --gcp-zone=us-east1-a
+        - --provider=gce
+        - --stage=gs://kubernetes-release-pull/ci/pull-kubernetes-e2e-gce-large-performance
+        - --test_args=--ginkgo.focus=\[Feature:Performance\] --minStartupPods=8 --gather-resource-usage=master --gather-metrics-at-teardown=master
+        - --timeout=540m
+        - --use-logexporter
+        # Bazel needs privileged mode in order to sandbox builds.
+        securityContext:
+          privileged: true
+        image: gcr.io/k8s-testimages/kubekins-e2e:v20180730-8b7ab3104-master
+        resources:
+          requests:
+            memory: "6Gi"
+
+  - name: pull-kubernetes-kubemark-e2e-gce-big
+    always_run: true
+    skip_report: false
+    max_concurrency: 12
+    skip_branches:
+    - release-1.10  # per-release image
+    - release-1.9  # per-release image
+    labels:
+      preset-service-account: "true"
+      preset-k8s-ssh: "true"
+      preset-bazel-scratch-dir: "true"
+      preset-dind-enabled: "true"
+    spec:
+      containers:
+      - image: gcr.io/k8s-testimages/kubekins-e2e:v20180730-8b7ab3104-master
+        args:
+        - --root=/go/src
+        - "--job=$(JOB_NAME)"
+        - "--repo=k8s.io/$(REPO_NAME)=$(PULL_REFS)"
+        - "--repo=k8s.io/release"
+        - "--service-account=/etc/service-account/service-account.json"
+        - "--upload=gs://kubernetes-jenkins/pr-logs"
+        - "--timeout=120"
+        - --scenario=kubernetes_e2e
+        - --
+        - --build=bazel
+        - --cluster=
+        - --env-file=jobs/env/ci-kubernetes-e2e-kubemark-common.env
+        - --env-file=jobs/env/ci-kubernetes-kubemark-gce-big.env
+        - --extract=local
+        - --gcp-master-size=n1-standard-4
+        - --gcp-node-size=n1-standard-8
+        - --gcp-nodes=7
+        - --gcp-project=k8s-presubmit-scale
+        - --gcp-zone=us-east1-b
+        - --kubemark
+        - --kubemark-nodes=500
+        - --provider=gce
+        - --stage=gs://kubernetes-release-pull/ci/pull-kubernetes-kubemark-e2e-gce-big
+        - --tear-down-previous
+        - --test=false
+        - --test_args=--ginkgo.focus=\[Feature:Performance\] --gather-resource-usage=true --gather-metrics-at-teardown=true
+        - --timeout=100m
+        # TODO(krzyzacy): Figure out bazel built kubemark image
+        #env:
+        # - name: KUBEMARK_BAZEL_BUILD
+        # value: "y"
+        # Bazel needs privileged mode in order to sandbox builds.
+        securityContext:
+          privileged: true
+        resources:
+          requests:
+            memory: "6Gi"
+  - name: pull-kubernetes-kubemark-e2e-gce-big
```

Comment:
Newline above this (here and elsewhere).

Reply:
done